gulpfile.js

```diff
@@ -66,40 +76,39 @@ var utilTasks = {
     mkdirsIfMissing(
         buildDir,
         buildDir + imagesDir,
-        buildDir + styleDir,
-        buildDir + jsDir
     );
   },
   'clean': function () {
     rmIfExists(buildDir);
   },
-  'lint-js': function () {
-    return gulp.src('src/**/*.js')
+  'lint-jsx': function () {
```

Comment:
these are fairly unnecessary, we usually consider both js and jsx to be js if that makes sense